app/controllers/code_review_copy_controller.rb

```diff
@@ -0,0 +1,13 @@
+class CodeReviewCopyController < ApplicationController
+  def create
+    code_review = CodeReview.find(params[:code_review][:id])
+    copy_code_review = code_review.duplicate_code_review(current_cohort)
+    if copy_code_review.save
+      flash[:notice] = "Code review successfully copied."
+      redirect_to cohort_code_reviews_path(current_cohort)
+    else
```

Comment:
When would a code review not be copied? If there is a possibility of failure, there should be an integration test to cover this path. Also, you should display the errors from the model, just like with any other form (instead of having a generic flash message.) Otherwise, take out the `else` condition.